docs/rules/indent.md

```diff
@@ -2,7 +2,7 @@
 
 This option validates a specific tab width for your code in block statements.
 
-There are several common guidelines, which indentation your code should have in nested blocks and statements, like:
+There are several common guidelines for which indentation your code should have in nested blocks and statements, like:
```

Comment:
I don't think this sentence makes sense even after your edit. I think this was supposed to be something like this:
```
There are several common guidelines which require indentation of nested blocks and statements
```